Makefile

```diff
@@ -32,11 +32,21 @@ MANAGER_IMAGE_PATCH := "apiVersion: apps/v1\
 \n      - image: <your image file>\
```

Comment:
`kind: Deployment` on patch

Reply:
Good catch. Not sure how my local copy of the output file was correct.